gpii/node_modules/flowManager/src/UserLogonStateChange.js

```diff
@@ -115,49 +142,82 @@ fluid.defaults("gpii.flowManager.userLogonStateChange.matchMakingStateChangeHand
 });
 
 
-// Incomplete grade - local FlowManager adds "gpii.flowManager.userLogonStateChange.matchMakingStateChangeHandler",
+// Incomplete grade - local FlowManager adds "gpii.flowManager.userLogonHandling.matchMakingStateChangeHandler",
 // whereas untrusted adds "gpii.flowManager.untrusted.stateChangeHandler"
 
-fluid.defaults("gpii.flowManager.userLogonStateChange.handler", {
+fluid.defaults("gpii.flowManager.proximityTrigger.handler", {
     gradeNames: ["kettle.request.http"],
     invokers: {
         handleRequest: {
-            funcName: "gpii.flowManager.userLogonStateChange.handleRequest",
-            args: ["{request}.req.params.userToken", "{flowManager}.lifecycleManager", "{that}"]
+            funcName: "gpii.flowManager.userLogonHandling.handleProximityTriggerRequest",
+            args: ["{that}", "{flowManager}.lifecycleManager", "{request}.req.params.userToken"]
         }
     }
 });
 
+/*
+ * Function to check whether a proximity trigger should bounce
+ *
+ * @param lifecycleManager {Object}: the lifecycleManager module
```

Comment:
We tend to say {Component} for these rather than {Object}. For the primitive types, we use capital letters e.g. {String}